apps/blockchain-mailer/src/store/envStore.ts

```diff
@@ -1,22 +1,67 @@
 class EnvStore {
+
+  get isTest(): boolean {
+    return process.env.NODE_ENV === 'test'
+  }
+
   get isDev(): boolean {
     return process.env.NODE_ENV === 'development'
   }
 
-  get isTest(): boolean {
-    return process.env.NODE_ENV === 'test'
+  get isStag(): boolean {
+    return process.env.NODE_ENV === 'staging'
   }
 
-  get ethereumNodeUrl(): string {
-    return process.env.ETHEREUM_NODE_URL || ''
+  get isProd(): boolean {
+    return process.env.NODE_ENV === 'production'
+  }
+
+  get envName() {
```

Comment:
Yea, it can be removed. I just afraid the filename would be too long (like `something.production.foo.bar`), but I think the full name is more generic.